src/components/Navbar/Navbar.css

```diff
@@ -5,27 +5,28 @@
 
 .navbar{
```

Comment:
Nav pills are overlapping screen's text or image, that's not a good user experience to me. You can give the background inset shadow or light color